libweasyl/libweasyl/alembic/versions/e0320dc462db_convert_frienduser_timestamps_to_.py

```diff
@@ -0,0 +1,52 @@
+"""convert frienduser timestamps to DateTime
+
+Revision ID: e0320dc462db
+Revises: 1d8412df7346
+Create Date: 2020-02-26 18:18:58.505000
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e0320dc462db'
+down_revision = 'cc2f96b0ba35'
+
+from alembic import op   # lgtm[py/unused-import]
+import sqlalchemy as sa  # lgtm[py/unused-import]
+
+
+def upgrade():
+    op.alter_column('watchuser', 'unixtime', server_default=None)
+    op.alter_column('watchuser', 'unixtime',
+               existing_type=sa.INTEGER(),
+               server_default=sa.func.now(),
+               type_=sa.DateTime(timezone=True),
+               existing_nullable=False,
+               postgresql_using="timestamp with time zone 'epoch' + (unixtime-18000) * interval '1 second';",
+               new_column_name='created_at')
+    op.alter_column('frienduser', 'unixtime', server_default=None)
+    op.alter_column('frienduser', 'unixtime',
+               existing_type=sa.INTEGER(),
+               server_default=sa.func.now(),
+               type_=sa.DateTime(timezone=True),
+               existing_nullable=False,
+               postgresql_using="timestamp with time zone 'epoch' + (unixtime-18000) * interval '1 second';",
```

Comment:
". Kind of surprised that this works with a semicolon.

```suggestion
               postgresql_using="to_timestamp(unixtime + 18000)",
```